config/keybow/config.exs

```diff
@@ -84,6 +84,37 @@ config :xebow, XebowWeb.Endpoint,
   url: [host: "xebow.local", port: 80],
   code_reloader: false
 
-config :xebow, settings_path: "/root/settings"
+config :xebow,
+  settings_path: "/root/settings",
+  layout: [
```

Comment:
Another approach to this would be `layout: Keybow.Layout` so that the type of layout is specified in the config, but the definition of the layout is in a dedicated module. This could also make it possible on `host` to support multiple layouts for testing or animation development.

Reply:
Oh good point; that would also remove the need for the `load_from_config` function.